staging/src/k8s.io/apimachinery/pkg/api/testing/roundtrip/roundtrip.go

```diff
@@ -216,10 +215,12 @@ func roundTripToAllExternalVersions(t *testing.T, scheme *runtime.Scheme, codecF
 		}
 		t.Logf("\tround tripping to %v %v", externalGVK, externalGoType)
 
+		t.Logf("\t\tjson")
```

Comment:
these output line were just for  debugging. I think you can drop them again (at least here. I would like to see an independent PR add them to the existing t.Logf's, i.e. to avoid extra lines in the test output).